Autopilot/TelemetryManager/telemetryStateClasses.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "telemetryStateManager.hpp"
+#include "telemetryManager.hpp"
+
+class initialMode: public telemetryState
+{
+    public:
+        void enter(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        void execute(telemetryManager* telemetryMgr);
+        void exit(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        static telemetryState& getInstance();
+
+    private:
+        initialMode(){}
+        initialMode(const initialMode& other);
+        initialMode& operator =(const initialMode& other);
+};
+
+class groundDataPathMode: public telemetryState
+{
+    public:
+        void enter(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        void execute(telemetryManager* telemetryMgr);
+        void exit(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        static telemetryState& getInstance();
+
+    private:
+        groundDataPathMode(){}
+        groundDataPathMode(const groundDataPathMode& other);
+        groundDataPathMode& operator =(const groundDataPathMode& other);
+};
+
+class errorReportMode: public telemetryState
+{
+    public:
+        void enter(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        void execute(telemetryManager* telemetryMgr);
+        void exit(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        static telemetryState& getInstance();
+
+    private:
+        errorReportMode(){}
+        errorReportMode(const errorReportMode& other);
+        errorReportMode& operator =(const errorReportMode& other);
+};
+
+class regularReportMode: public telemetryState
+{
+    public:
+        void enter(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        void execute(telemetryManager* telemetryMgr);
+        void exit(telemetryManager* telemetryMgr){(void) telemetryMgr;}
+        static telemetryState& getInstance();
+
+    private:
+        regularReportMode(){}
+        regularReportMode(const regularReportMode& other);
+        regularReportMode& operator =(const regularReportMode& other);
+};
```

Comment:
Same thing here with the newline!

Reply:
@Pop0097 Nah defines are a C thing.... we should be moving away from them in CPP files and instead using const since defines don't offer any type protection as it swaps out literals